.github/workflows/gosec.yml

```
@@ -28,8 +28,12 @@ jobs:
       - name: Run Gosec Security Scanner
         uses: securego/gosec@master
         with:
-          args: '-no-fail -fmt sarif -out results.sarif ./...'
+          args: '-fmt sarif -out gosec.sarif ./...'
+      
+      - name: Print SARIF file
+        run: cat gosec.sarif
+
       - name: Upload SARIF file
         uses: github/codeql-action/upload-sarif@v1
         with:
-          sarif_file: results.sarif
+          sarif_file: gosec.sarif
```

@djaglowski Strange, this seems to be a [GoSec issue](https://github.com/securego/gosec/issues/582#issuecomment-790452394), that should have already been fixed. From your pasted output, the GoSec Docker image doesn't seem to be outputting correctly. 

For now, perhaps we could remove the SARIF file generation/upload to Github altogether: 
```suggestion
          args: './...'
```
I will submit a PR directly to update this later, or an issue if it is in fact a problem with GoSec.